authentication/openshift/options.go

```diff
@@ -138,7 +138,7 @@ func (s *DelegatingAuthenticationOptions) getRequestHeader() (*RequestHeaderAuth
 	return nil, fmt.Errorf("no request header config")
 }
 
-func (s *DelegatingAuthenticationOptions) newTokenAccessReview() (authenticationclient.TokenReviewInterface, error) {
+func (s *DelegatingAuthenticationOptions) newAuthenticationClient() (authenticationclient.AuthenticationV1Interface, error) {
```

Comment:
Can you link a k8s PR that explains why we switch the interface here?

Reply:
v0.21.1: https://pkg.go.dev/k8s.io/apiserver@v0.21.1/pkg/authentication/authenticatorfactory#DelegatingAuthenticatorConfig

v0.26.1: https://pkg.go.dev/k8s.io/apiserver/pkg/authentication/authenticatorfactory#DelegatingAuthenticatorConfig

Reply:
:+1: